DFC.App.DiscoverSkillsCareers/Views/Assessment/ReferenceSent.cshtml

```diff
@@ -6,7 +6,7 @@
             <p>A text message has been sent to @TempData.Peek("Telephone")</p>
             <p>Click on the link in the text message to return to your assessment.</p>
             <a asp-controller="assessment" class="govuk-button" asp-action="return">Return to assessment</a>
-            <p><a href="/discover-your-skills-careers">Back to start</a></p>
+            <p><a href="/discover-your-skills-and-careers">Back to start</a></p>
```

Comment:
Can you get "discover-your-skills-and-careers" from @RouteName.Prefix

Reply:
changed.